Subject: [xf86-video-vesa] make packet sysroot aware

The xf86-video-vesa packet tries to include headers from
$(prefix)/include, which isn't correct. Usually, when building
distributions, packets are configured with --prefix=/usr and installed
with SYSROOTS=....

Signed-off-by: Luotao Fu <l.fu@pengutronix.de>
Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xf86-video-vesa-2.2.1/configure.ac
===================================================================
--- xf86-video-vesa-2.2.1.orig/configure.ac
+++ xf86-video-vesa-2.2.1/configure.ac
@@ -89,7 +89,7 @@ fi
 AM_CONDITIONAL(XSERVER_LIBPCIACCESS, test "x$XSERVER_LIBPCIACCESS" = xyes)
 
 CFLAGS="$CFLAGS $XORG_CFLAGS "' -I$(top_srcdir)/src'
-INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src -I$(prefix)/include'
+INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src '"-I${SYSROOT}"'$(prefix)/include'
 AC_SUBST([CFLAGS])
 AC_SUBST([INCLUDES])
 
